index.html

```diff
@@ -33,10 +33,11 @@ <h1>I’m Marcelo
             <span>LET’S CONNECT</span>
         </div>
         <ul class="social-icons">
-            <li><img src="/PersonalPortfolio/imagenes/twiter.png" alt="twiter"></li>
-            <li><img src="/PersonalPortfolio/imagenes/Linkedin icon.png" alt="Linkedin"></li>
-            <li><img src="/PersonalPortfolio/imagenes/git.png" alt="git-ic"></li>
-            <li><img src="/PersonalPortfolio/imagenes/chill.png" alt="chill-ico"></li>
+            <li><a><img src="/imagenes/twiter.png" alt="twiter"></a></li>
+            <li><a><img src="/imagenes/Linkedin icon.png" alt="Linkedin"></a></li>
+            <li><a><img src="/imagenes/git.png" alt="git-ic"></a></li>
+            <li><a><img src="/imagenes/chill.png" alt="chill-ico"></a></li>
+            <li><a><img src="/imagenes/m.png" alt="m-logo"></a></li>
         </ul>
     </div>
 </section>
```

Comment:
Great job @marchelazo 🤝

- [ ] Kindly make sure your design has a background image. You may remove the `/PersonalPortfolio` from your path to make the background image take effect.
![background image](https://user-images.githubusercontent.com/63917725/191856496-f20511b0-bd9b-42ed-b571-60f9bbdade6a.PNG)